CMakeLists.txt

```diff
@@ -4,33 +4,80 @@ project(ccpp_framework
         VERSION 5.0.0
         LANGUAGES Fortran)
 
+enable_language(C CXX)
+include(FetchContent)
+include(cmake/ccpp_capgen.cmake)
+set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${PROJECT_SOURCE_DIR}/cmake")
+
+
 #------------------------------------------------------------------------------
 # Set package definitions
 set(PACKAGE "ccpp-framework")
 set(AUTHORS "Dom Heinzeller" "Grant Firl" "Mike Kavulich" "Dustin Swales" "Courtney Peverley")
```

Comment:
Should we add more people here? Those who regularly contribute code and are on the CCPP framework dev team (@mwaxmonsky for example???)